Gemfile

```diff
@@ -66,6 +72,7 @@ group :test do
   gem 'rubocop-rspec', require: false
   gem 'selenium-webdriver'
   gem 'simplecov', require: false
+  gem 'vcr'
```

Comment:
I am not keen on this gem - it creates an overhead that we don't need at the moment I think. Rerecording of cassettes etc can be a real headache.